### DISCUSSION TITLE
my github page doesn't redirect to index.html

### DISCUSSION THREAD

**Discussion by ghost** [community/community#22438] [Other Feature Feedback, Questions, & Ideas]
<p>i already add index.html on my github page repository</p>
<p>but the page didn’t show up</p>
<p>github repository:</p>
<p><a href="https://github.com/ZeVoidah/ZeVoidah.github.io" rel="nofollow noopener">https://github.com/ZeVoidah/ZeVoidah.github.io</a></p>
<p>github page:</p>
<p><a href="https://ZeVoidah.github.io" rel="nofollow noopener">https://ZeVoidah.github.io</a></p>
<p>this happens when i add a cname and delete it.</p>
<p>help me lol.</p>

**Comment by ThirdEyeRose** [comment]
<p>Hi <a class="mention" href="/zevoidah">@zevoidah</a>,</p>
<p>This seems to be working now. Would you mind sharing with us what was wrong and what you did to fix it?</p>
<p>Thanks!</p>

**Comment by ghost** [comment]
<p>it shows on my brother’s pc but it didn’t show on my pc so i think there’s a problem with my stupid old computer lol, or my google chrome browser blocks it.</p>
<p>it works when i go to this link <a href="https://zevoidah.github.io/index" rel="nofollow noopener">https://zevoidah.github.io/index</a></p>
<p>but it didn’t work to this main link <a href="https://zevoidah.github.io/" rel="nofollow noopener">https://zevoidah.github.io/</a></p>
<p>also i fix it with deleting the repository and make them again.</p>

**Comment by sidharathkhanna29** [comment, marked as answer]
<p>Hey I think, may be i am late for the reply but you not need to delete your repository for that. Just try deleting your Cookies and Cache from your browser.</p>
<p>Actually i did the same mistake for my page and just wanted no one else unnecessarily deletes his/her repository for same. </p>

**Comment by shreyashkumar** [comment]
<p>Even I am facing the same issue and I tried deleting my cookies and cache but the problem persists.</p>
<p>My website url: <a href="https://shreyashkumar.github.io" rel="nofollow noopener">https://shreyashkumar.github.io</a>.</p>
<p>This is not working rather I have to type in my browser</p>
<p><a href="https://shreyashkumar.github.io/index%20.html" rel="nofollow noopener">https://shreyashkumar.github.io/index%20.html</a></p>
<p>My github repository URL:<a href="https://github.com/shreyashkumar/shreyashkumar.github.io" rel="nofollow noopener">https://github.com/shreyashkumar/shreyashkumar.github.io</a></p>

**Comment by shreyashkumar** [comment]
<p>Ohh! It’s working now.</p>
<p>The problem was due to there was an extra space after index</p>
<p>like index .html which was suppose to be index.html.</p>
<p>It’s working fine now.</p>

**Comment by alexlovell149** [comment]
<p>I created a github pages on my portfolio and just redirects to the README? Does anyone know how to fix this? I tried deleting the repository and remaking it</p>

**Comment by nadiajoyce** [comment]
<p>Hi <a class="mention" href="/alexlovell149">@alexlovell149</a>,</p>
<p>It sounds like you may be experiencing a different problem than is discussed in this thread. I’d recommend opening a new topic in the <a href="/t5/GitHub-Pages/bd-p/pages">GitHub Pages</a> board to ask for help. I’d also recommend providing a bit more information, like what steps you’ve already tried to resolve the problem, and if your repository is public, providing a link to that can be really helpful, too.</p>
<p>Cheers!</p>

**Comment by Versifiction** [comment]
<p>I have the same issue, is it working for you now?</p>

**Comment by nadiajoyce** [comment]
<p>Hi <a class="mention" href="/versifiction">@versifiction</a> What problem are you experiencing? The same as the OP? If so, I’d recommend checking out the Accepted Solution, as well as some of the other things that users tried in this thread for troubleshooting tips. If you are experiencing a problem other than the one the original post talks about, I’d recommend opening a whole new Topic in order to ask your question. You’ll be more likely to get the proper help that way.</p>
<p>Thanks!</p>

**Comment by Versifiction** [comment]
<p>My index.html is in a folder “src”, so my Github page is not correct, but I can’t move the index.html from the folder to the root</p>
<p>(I made a topic for this but no answer yet)</p>

**Comment by rebeccapeltz** [comment]
<p>I’ve got the same problem - <a href="https://suwebdev.github.io/user-science-blog/" rel="nofollow noopener">https://suwebdev.github.io/user-science-blog/</a> doesn’t work, but <a href="https://suwebdev.github.io/user-science-blog/index.html" rel="nofollow noopener">https://suwebdev.github.io/user-science-blog/index.html</a> does</p>
<p>Tested on chrome and  safari.</p>
<p>I tried clearing cookies.</p>
<p>Rebecca</p>

**Comment by nadiajoyce** [comment]
<p>Hi <a class="mention" href="/rebeccapeltz">@rebeccapeltz</a>,</p>
<p>I just took a look and both of those URLs appear to be working for me. Are you still experiencing this issue?</p>

**Comment by shahzadadil** [comment]
<p>Your problem might be because there is also a Readme.md file present. Try to delete .md file and verify</p>

**Comment by joseph-belmonte** [comment]
<p><a class="mention" href="/shahzadadil">@shahzadadil</a>  this fixed it for me; thanks! </p>

**Comment by ManoharYeluri** [comment]
<p>I too had faced this problem…</p>
<p>There are multiple files in my plain html, css &amp; javascript repo. obviously, thers is   <strong>index.html</strong>. but, along with that, i named my main js file as " <strong>index.js</strong>". So, it didn’t open index.html by default until i changed my js file name to something other than index.</p>

**Comment by camyyssa** [comment]
<p>Renamed <code>readme.md</code> to <code>README.md</code> which fixed the issue. </p>
<p>I had <code>index.html</code> in the root of my repository which Github pages wasn’t redirecting to. (<code>x.github.io/page</code> didn’t work, <code>x.github.io/page/index.html</code> worked)</p>

**Comment by nabilateuse** [comment]
<p>may beThe problem due to there was an extra space after index</p>
<p>which was suppose to be index.html.</p>

**Comment by nabilateuse** [comment]
<p>may beThe problem due to there was an extra space after index</p>
<p>which was suppose to be index.html.</p>
<blockquote>
<p><a class="mention" href="/zevoidah">@zevoidah</a> wrote:</p>
</blockquote>
<blockquote>
<p>i already add index.html on my github page repository</p>
</blockquote>
<blockquote>
<p>but the page didn’t show up</p>
</blockquote>
<blockquote>
<p> </p>
</blockquote>
<blockquote>
<p>github repository:</p>
</blockquote>
<blockquote>
<p><a href="https://github.com/ZeVoidah/ZeVoidah.github.io" rel="nofollow noopener">https://github.com/ZeVoidah/ZeVoidah.github.io</a></p>
</blockquote>
<blockquote>
<p> </p>
</blockquote>
<blockquote>
<p>github page:</p>
</blockquote>
<blockquote>
<p><a href="https://ZeVoidah.github.io" rel="nofollow noopener">https://ZeVoidah.github.io</a></p>
</blockquote>
<blockquote>
<p> </p>
</blockquote>
<blockquote>
<p>this happens when i add a cname and delete it.</p>
</blockquote>
<blockquote>
<p>help me lol.</p>
</blockquote>

**Comment by gbgabiola** [comment]
<p>this is right, even at the time of this writing (year 2020).</p>
